README.md

```diff
@@ -35,16 +35,23 @@ console.log(ObjCodes.Baseline)
 
 ```typescript
 // TypeScript type definitions are bundled
-import {TObjCode} from 'workfront-objcodes'
+import {TObjCode, OpTask} from 'workfront-objcodes'
 
 // TS2322: Type '"FOO"' is not assignable to type 'TObjCode' 
-const myObjCode: TObjCode = 'FOO'
+const myObjCode = 'FOO' as const
```

Comment:
I don't see connection between this line and comment above. Why can't we leave this line unchanged?

Reply:
Ah, I mean, if you put the `: TObjCode` it should also be an error at that line since the value isn't part of the union, but typically you don't define the const as the type like this, but you do define functions or objects that expect a `TObjCode` as argument or property value.

This would be one case though where you do a type cast too.

```ts
const data = someApiFuncCall();

function someObjCodeFunc(code: TObjCode) {}

someObjCodeFuncCall(data.objCode as TObjCode)
```

I guess I just wanted to show the most common usage of the type.

Reply:
I see, then it is better to move comment describing error to the line where error actually happens - `useObjCode(myObjCode);`.